clients/lightrider/lightrider-ui-features.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global ReportUIFeatures */
+
+/**
+ * Extends ReportUIFeatures for a Lightrider viewer to get html from a browserified ReportGenerator.
+ */
+class LightriderUIFeatures extends ReportUIFeatures {
+  /**
+   * Uses ReportGenerator to create the html that recreates this report.
+   * @return {string}
+   * @override
+   */
+  getReportHtml() {
+    // @ts-ignore ReportGenerator is put on window by the host page.
+    return window.ReportGenerator.generateReportHtml(this.json);
```

Comment:
in the long term i'm interested in changing `ReportUIFeatures.getReportHtml()` to be this instead. and we'd always ship reportgenerator inside of the HTML report. 

but that requires some work. so it'll wait.